packages/compartment-mapper/src/import-hook.js

```diff
@@ -70,13 +71,16 @@ export const makeImportHookMaker = (
     _packageName,
     parse,
     shouldDeferError,
+    compartments,
   ) => {
     // per-compartment:
     packageLocation = resolveLocation(packageLocation, baseLocation);
     const packageSources = sources[packageLocation] || Object.create(null);
     sources[packageLocation] = packageSources;
-    const compartment = compartments[packageLocation] || {};
-    const { modules = Object.create(null) } = compartment;
+    const compartmentDescriptor = compartmentDescriptors[packageLocation] || {};
```

Comment:
It might be smart to leave a `const compartmentName = packageLocation;` line above here just to make clear that the equivalence is intentional in this layer of the architecture.

Reply:
```js
    packageLocation = resolveLocation(packageLocation, baseLocation);
    ```
    surprised the linter tolerates this